pkgs/by-name/gl/glog/package.nix

```
@@ -19,6 +20,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-+nwWP6VBmhgU7GCPSEGUzvUSCc48wXME181WpJ5ABP4=";
   };
 
+  postPatch = ''
+    substituteInPlace src/logging_unittest.cc \
+      --replace-warn "/usr/bin/true" "${coreutils}/bin/true"
```

With just `coreutils` you'd be adding a dependency on the hostPlatform build of coreutils, which does not make sense for most cross setups.
```suggestion
      --replace-warn "/usr/bin/true" "${pkgsBuildHost.coreutils}/bin/true"
```